modules/nw-dual-stack-convert.adoc

```diff
@@ -0,0 +1,81 @@
+[id="nw-dual-stack-convert_{context}"]
+= Converting to a dual network stack cluster network
+
+As a cluster administrator, you can convert your single stack cluster network to a dual-stack cluster network.
+
+[NOTE]
+====
```

Comment:
The NOTE receives more of a highlight than might be necessary.  If you think you have a place for a bullet in a concepts area, I'd move this note into a bullet in that concepts area.

Reply:
@mikemckiernan, yeah, this really needs some kind of _realities of dual-stack networking_ section somewhere.